todo_app/app.py

```diff
@@ -102,7 +138,7 @@ def callback():
         user_response = requests.get('https://api.github.com/user', headers=user_headers)
 
         user_info = user_response.json()
-        print('user_info', user_info)
+        app.logger.debug('user_info', user_info)
```

Comment:
As per the above comments, this doesn't actually show the info to me as it is, but if we do log it there's quite a lot of account info that I don't necessarily think we want to end up in ours logs!